trunk/src/test/sql/uestcojtest.sql

```diff
@@ -87,6 +87,7 @@ CREATE  TABLE IF NOT EXISTS `uestcojtest`.`problem` (
   `javaMemoryLimit` INT NOT NULL DEFAULT 65535 ,
   `dataCount` INT NOT NULL DEFAULT 0 ,
   `difficulty` INT NOT NULL DEFAULT 1 ,
+  `type` INT NOT NULL DEFAULT 0 ,
```

Comment:
Also add this line in  `trunk/src/main/sql/uestcoj.sql`

And create a patch sql in `trunk/src/main/sql/patches/` for developer